homeassistant/components/hyperion/sensor.py

```diff
@@ -0,0 +1,230 @@
+"""Sensor platform for Hyperion."""
+from __future__ import annotations
+
+import functools
+import logging
+from typing import Any
+
+from hyperion import client
+from hyperion.const import (
+    KEY_COMPONENTID,
+    KEY_ORIGIN,
+    KEY_OWNER,
+    KEY_PRIORITIES,
+    KEY_PRIORITY,
+    KEY_RGB,
+    KEY_UPDATE,
+    KEY_VALUE,
+    KEY_VISIBLE,
+)
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_connect,
+    async_dispatcher_send,
+)
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import slugify
+
+from . import (
+    get_hyperion_device_id,
+    get_hyperion_unique_id,
+    listen_for_instance_updates,
+)
+from .const import (
+    CONF_INSTANCE_CLIENTS,
+    DOMAIN,
+    HYPERION_MANUFACTURER_NAME,
+    HYPERION_MODEL_NAME,
+    SIGNAL_ENTITY_REMOVE,
+    TYPE_HYPERION_SENSOR_VISIBLE_PRIORITY,
+)
+
+_LOGGER = logging.getLogger(__name__)
+SENSORS = [TYPE_HYPERION_SENSOR_VISIBLE_PRIORITY]
+
+
+def _sensor_unique_id(server_id: str, instance_num: int, suffix: str) -> str:
+    """Convert a sensor to a unique_id."""
+    return get_hyperion_unique_id(
+        server_id,
+        instance_num,
+        slugify(f"{suffix}"),
```

Comment:
... and here you'd include the base in the unique_id, 

```suggestion
        slugify(f"{TYPE_HYPERION_SENSOR_BASE}_{suffix}"),
```